### PR TITLE
Adding the sphero rvr sdk so that this docker stuff can build

### PR DESCRIPTION
**Pull request by ParmaJonEman**
#16 

Added the ability to specify the ROS_DOMAIN_ID with a -r parameter (default 0) and set some properties in the docker compose so that the ROS2 topics are visible anywhere on the network.

### CODE REVIEW

File: ros2_ws/src/rvr_node/rvr_node/rvr_node_app.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 # import sphero sdk packages
 from sphero_sdk_wrapper.sphero_rvr_interface import SpheroRvrInterface, initialize_rvr_interface, initialize_rvr_sdk
 
+import os
```
Comment (esimkowitz) [inline ros2_ws/src/rvr_node/rvr_node/rvr_node_app.py]: Why is this necessary?

**Review by esimkowitz — COMMENTED** [review]
Looks good, left one comment

File: ros2_ws/src/rvr_node/rvr_node/rvr_node_app.py (hunk 1 above)
Comment (ParmaJonEman) [reply ros2_ws/src/rvr_node/rvr_node/rvr_node_app.py]: It's not, that was a mistake. Just fixed it.

**Review by esimkowitz — APPROVED** [review]
